### PR TITLE
Listing tags

### PR DESCRIPTION
**Pull request by antgonza**
ToDo:
- [x] make tests pass
- [x] test in qiita-test env

### CODE REVIEW

**Comment by antgonza** [comment]
This should be ready for review after https://github.com/biocore/qiita/pull/2088 is merged. This code has been deployed in the test env and seems to be working fine. Note that I had to update some js and css libraries

Example of functionality, see how the "regular" search and the tags interact nicely:
tags

File: qiita_pet/handlers/study_handlers/tests/test_base.py [hunk 1]
```diff
@@ -59,5 +59,37 @@ def test_get(self):
...
+class TestStudyGetTags(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/study/get_tags/')
+        exp = ('{"status": "success", "message": "", "tags": '
+               '{"admin": [], "user": []}}')
+        self.assertEqual(response.code, 200)
+        self.assertEqual(response.body, exp)
+
+
+class TestStudyTags(OauthTestingBase):
```
Comment (wasade) [inline qiita_pet/handlers/study_handlers/tests/test_base.py]: should there be negative test cases?

**Comment by antgonza** [comment]
Thanks @wasade note that this is based on: https://github.com/biocore/qiita/pull/2088 ....

**Comment by wasade** [comment]
Should I not review this then?

**Comment by josenavas** [comment]
@antgonza can you pull from upstream/master?

**Comment by antgonza** [comment]
pull from upstream/master ready. Note that currently we are using tag-it also to display it and it looks like this:
screen shot 2017-03-27 at 4 55 55 pm
but I could simplify things so it looks like this:
screen shot 2017-03-27 at 4 56 00 pm
I really don't have a preference but wanted to ask ...

**Comment by antgonza** [comment]
@wasade, @josenavas, @ElDeveloper, could you review?

**Comment by josenavas** [comment]
Thanks @antgonza code looks good!

Answering your last question - I think either looks fine, I don't have a strong preference. What I don't really like is the position of the filter. i.e. I don't like it showing inside the table. Is it easy to have on top of the table (i.e. above the headers?). The reason why I don't like it there is because it looks like only applies to the top table, but it should be applied to both tables (private and public studies).

Do you think that change can be done easily?

**Comment by antgonza** [comment]
Not difficult at all, we have 3 options:
1. what we have now
2. this: screen shot 2017-03-29 at 11 42 40 am
3. or this: screen shot 2017-03-29 at 11 42 13 am

**Comment by ElDeveloper** [comment]
Option 2 looks great 👍

**Comment by josenavas** [comment]
My vote goes to 2 - I like when things are modular and can be moved around!!! :D

**Comment by antgonza** [comment]
This is the pushed version: screen shot 2017-03-29 at 11 50 07 am

**Comment by wasade** [comment]
merge?

**Comment by ElDeveloper** [comment]
:shipit:

**Pull request merged**